ReactNative/ReactTwitter/__tests__/core/service/percent-encoder.test.js

```diff
@@ -0,0 +1,17 @@
+jest.unmock('../../../core/service/percent-encoder.js')
+var PercentEncoder = require('../../../core/service/percent-encoder.js')
+
+describe('PercentEncoder', () => {
+  describe('encode', () => {
+    it('when encoding', () => {
```

Comment:
You should really use the proper jasmine way of doing this. The pattern is:

```
it('should encode a special character', () => {
    // ...
})

it('should encode only special characters', () => {
    // ...
})

// etc
```

Don't write one `it` for multiple tests, or it'll break badly when (not if 👅) your code fails.